docs/userguide/service/working-with-istio-on-non-flat-network.md

```diff
@@ -12,11 +12,13 @@ not direct connectivity between the pods in all clusters.
 ![Istio on Karmada-different-network](../../resources/userguide/service/istio/istio-on-karmada-different-network.png)
 
 ***
```

Comment:
Can you remind me why change this `***` to `---`, seems they are both used to add a new line.

I'm ok with it, it seems this one forgot to change.

Reply:
It is just because of markdown formatter in my editor.

Reply:
Is there a good reason to do this? Is there some explanation from your editor?

I'm not sure if another editor would change this back :)